keras_nlp/models/bert/bert_tasks.py

```diff
@@ -123,44 +204,70 @@ def from_preset(
     ):
         """Create a classification model from a preset architecture and weights.
 
+        By default, this method will automatically create a `preprocessor`
+        layer to preprocess raw inputs during `fit()`, `predict()`, and
+        `evaluate()`. If you would like to disable this behavior, pass
+        `preprocessor=None`.
```

Comment:
Since we can't use the same flag here as the base class (`include_preprocessing`), I appreciate the note!

Should we also include an example of specifying the preprocessor with overrides?
```python
preprocessor=BertPreprocessor.from_preset(
    "bert_base_uncased_en",
    sequence_length=128,
)
```

Reply:
done!